### PR TITLE
CircleCI + staging tables

### PR DESCRIPTION
**Pull request by fivetran-kristin**
- Add CircleCI integration tests 
- staging table set up
- include all default columns in table
- Create pass through column variables

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,14 +28,27 @@ By default, this package will run using your target database and the `salesforce
...
 ...
-config-version: 2
-
 vars:
   salesforce_source:
     salesforce_database: your_database_name
     salesforce_schema: your_schema_name
 ```
 
+This package includes all source columns define in the `generate_columns.sql` macro. To add additional columns to this package, do so using our pass-through column variables. This is extremely useful if you'd like to include custom fields to the package.
```
Comment (fivetran-jamie) [inline README.md:37]: typo around "define" in the first sentence

File: models/stg_salesforce__user_role.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+        {% endif %}
+
+    from source
+
+)
+
+select * 
+from renamed
+where not _fivetran_deleted
```
Comment (fivetran-jamie) [inline models/stg_salesforce__user_role.sql:31]: might wanna use `where not coalesce(_fivetran_deleted, false)`

**Review by fivetran-jamie — COMMENTED** [review]
looks good to me!  just made a tiny suggestion regarding coalescing _fivetran_deleted, since that sometimes causes issues for us

File: models/stg_salesforce__user_role.sql (hunk 2 above)
Comment (fivetran-kristin) [reply models/stg_salesforce__user_role.sql]: Great catch -- updated!

**Review by fivetran-jamie — APPROVED** [review]
looks good!!

**Pull request merged**